controllers/storagecluster/reconcile.go

```diff
@@ -159,14 +161,24 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 		return reconcile.Result{}, err
 	}
 
+	var err error
+	availableCrds, err := statusutil.MapCRDAvailability(ctx, r.Client, statusutil.GetCrds()...)
+	if err != nil {
+		r.Log.Error(err, "error fetching the CRDs")
+		return reconcile.Result{}, err
+	}
+	if !reflect.DeepEqual(availableCrds, r.AvailableCrds) {
+		r.Log.Info("CustomResourceDefinitions created/deleted. Restarting process.")
+		os.Exit(42)
```

Comment:
Can we please have the exit code in a constant with a name that explains the usage? 
In addition, a comment about this line will help

Reply:
Added it with https://github.com/red-hat-storage/ocs-operator/pull/2745